geode-core/src/main/java/org/apache/geode/internal/cache/InternalRegion.java

```diff
@@ -449,6 +450,8 @@ default void unlockWhenRegionIsInitializing() {
 
   CachePerfStats getRegionPerfStats();
 
+  void handleInterestEvent(InterestRegistrationEvent event);
```

Comment:
Do you want to make it `public`?

Reply:
Yes, I think it's ok to make this public. Users can already use Reflection to invoke non-public methods on LocalRegion anyway. My overall method is this:

1. switch over to using interfaces for all dependencies and interactions between classes
2. use dependency injection to make unit testing easy
3. review the interfaces and break up larger ones to better represent the actual dependencies/interactions between packages or components